runtime/neumann/src/lib.rs

```diff
@@ -929,6 +929,11 @@ impl pallet_automation_price::Config for Runtime {
 	type UniversalLocation = UniversalLocation;
 	type SelfParaId = parachain_info::Pallet<Runtime>;
 	type EnsureProxy = AutomationEnsureProxy;
+
+	// how many oracle can be authorized to update price for a pair
+	type MaxAuthorizedOracleWallet = ConstU32<5>;
+	// How many item we can put into the same price update extrinsic
+	type MaxBatchPriceUpdate = ConstU32<100>;
```

Comment:
These are good ideas. The variable names are self-explanatory. Since they are new, we will need to document them along with other new events or chain configs on docs.oak.tech. 

I suggest us to keep it simple for now since we will be using a single oracle for the next 3-6 months, because these two parameters will need a set of unit tests to ensure the edge cases or errors are handled correctly